### ISSUE TITLE
Product description causes horizontal scrolling

### ISSUE THREAD

**Issue by ravenokko** [shopware/shopware#20378]
Product name/description/properties triggers horizontal scrolling which does not meet the success criteria for [SC 1.4.10](https://www.w3.org/WAI/WCAG22/Understanding/reflow)
Properties still cutoff even when horizontally scrolled

<img width="25%" height="25%" alt="Image" src="https://github.com/user-attachments/assets/33b94bf9-5c94-48e5-b329-a5a026d79310" />
<img width="25%" height="25%" alt="Image" src="https://github.com/user-attachments/assets/64cee05c-a2a1-4fa9-bb18-eb1997d8f38e" />

### How to reproduce

In Firefox go to Preferences > General > Zoom
- Ensure default Zoom is set to 100%
- check Zoom text only

Create shop with demodata

Visit storefront homepage
- Set viewport to 320px wide
- Zoom text to 200%
- Open product detail > description

**Comment by ravenokko** [comment]
related: https://github.com/shopware/shopware/issues/20380
